content/en/docs/concepts/cluster-administration/kube-state-metrics.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Metrics for Kubernetes object state
+content_type: concept
+weight: 75
+description: >-
+   kube-state-metrics, an add-on agent to generate and expose cluster-level metrics.
+---
+
+The state of Kubernetes objects in the Kubernetes API can be exposed as metrics.
+An add-on agent called [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) can connect to the Kubernetes API server and expose a HTTP endpoint with metrics generated from the state of individual objects in the cluster.
+It exposes various information about the state of objects like labels and annotations, startup and termination times, status or the phase the object currently is in.
+For example, containers running in pods create a `kube_pod_container_info` metric.
+This includes the name of the container, the name of the pod it is part of, the namespace the pod is running in, the name of the container image, the ID of the image, the spec of the image, the ID of the running container and the ID of the pod as labels.
```

Comment:
For clarity, try:
```suggestion
This includes the name of the container, the name of the pod it is part of, the
{{< glossary_tooltip text="namespace" term_id="namespace" >}} the pod is running in,
the name of the container image, the ID of the image, the spec of the image, the ID of
the running container and the ID of the pod. These details are exposed as OpenMetrics
labels.
```

Is “the spec of the image” a fragment of JSON? It's not obvious from the text what that label's value might be.

Reply:
It's the image name from the spec https://github.com/kubernetes/kube-state-metrics/blob/main/internal/store/pod.go#L154 and the rest comes from the container status.